client/src/shared/components/Radiobutton/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { Fragment } from 'react';
+
+export interface RadioButtonProps {
+  label?: string;
+  options: string[];
+  alignment?: string;
+  classname?: string;
+  onClickEvent: () => unknown;
```

Comment:
- return the selected option.
```suggestion
  onClickEvent: (query: string) => void;
```